.buildbot.sh

```diff
@@ -4,22 +4,49 @@ set -e
 
 find . -iname "*.c" -o -iname "*.h" -o -iname "*.cpp" -o -iname "*.hpp" | xargs ~/cheri/output/sdk/bin/clang-format --dry-run -Werror
 
+repodir=$(pwd)
+
 export CHERI=$HOME/cheri/output/sdk
 export LD_LIBRARY_PATH=$CHERI/lib
 export CC=$CHERI/bin/clang 
 export LLVM_EXTRACT=$CHERI/bin/llvm-extract
+export AR=$CHERI/bin/llvm-ar 
+export RANLIB=$CHERI/bin/llvm-ranlib
+export LLVM_LINK=$CHERI/bin/llvm-link
 
 make CC=$CC CXX=$CHERI/bin/clang++ LLVM_CONFIG=$CHERI/bin/llvm-config -j3
 
-cd tests
-
-if ! [ -x "$(command -v cargo)" ]; then
-	curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > rustup.sh
-	sh rustup.sh --default-host x86_64-unknown-linux-gnu \
-		--default-toolchain stable \
-		--no-modify-path \
-		--profile minimal \
-		-y
-	source $HOME/.cargo/env
-fi
-cargo test
+pushd tests
+	if ! [ -x "$(command -v cargo)" ]; then
```

Comment:
I don't think these lines should be indented?

Reply:
It allows the ability to visually determine which things run in a different directory then the original one.